code/modules/clothing/masks/gasmask.dm

```diff
@@ -186,12 +188,22 @@ GLOBAL_LIST_INIT(clown_mask_options, list(
 /obj/item/clothing/mask/gas/atmos/plasmaman
 	starting_filter_type = /obj/item/gas_filter/plasmaman
 
-/obj/item/clothing/mask/gas/atmos/captain
-	name = "captain's gas mask"
-	desc = "Nanotrasen cut corners and repainted a spare atmospheric gas mask, but don't tell anyone."
+/obj/item/clothing/mask/gas/captain
+	name = "captain's rebreather"
+	desc = "A state of the art mask, with a dual purpose slot for either a gas filter or narcotics."
+	actions_types = list(/datum/action/item_action/adjust)
+	flags_inv = HIDEFACIALHAIR | HIDEFACE | HIDESNOUT
+	flags_cover = MASKCOVERSMOUTH | PEPPERPROOF
 	icon_state = "gas_cap"
 	inhand_icon_state = "gasmask_captain"
+	armor_type = /datum/armor/gas_atmos
 	resistance_flags = FIRE_PROOF | ACID_PROOF
+	has_fov = FALSE
+	max_filters = 1
```

Comment:
```suggestion
	max_filters = 2
```

Why not two filter slots! One for gas filtering, the other for cigars. Perfectly in character.

Reply:
to put a cigar in, you need to have NO filters on the masks, which is why i removed the extra 2 in the first place